shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -35,7 +43,28 @@
  */
 @ThreadSafe
 public final class DistributedLoadCommand extends AbstractFileSystemCommand {
-  private static final String REPLICATION = "replication";
+  public static final Option REPLICATION =
+      Option.builder()
+          .longOpt("replication")
+          .required(false)
+          .hasArg(true)
+          .desc("number of replicas to have for each block of the loaded file")
+          .build();
+  public static final Option THREAD_OPTION =
```

Comment:
nit: `THREADS_OPTION` to be consistent

Reply:
Do you mean const value like C language?